syft/cataloging/license.go

```diff
@@ -27,7 +23,7 @@ type LicenseConfig struct {
 
 func DefaultLicenseConfig() LicenseConfig {
 	return LicenseConfig{
-		IncludeContent: LicenseContentExcludeAll,
+		IncludeContent: LicenseContentIncludeUnknown,
```

Comment:
I thought from the discussion we said the default should be to never include content but instead the licenses show up as `UNKNOWN`, otherwise users will potentially start getting very large SBOMs.

Reply:
I encouraged this change, however after thinking about it more, I agree with you and chris on this: we should leave this as ExcludeAll. The revised thinking: we should treat #3088 as a bug... meaning we should have excluded contents to begin with. Preserving the preexisting behavior isn't necessary.